x-pack/metricbeat/module/azure/_meta/docs.asciidoc

```diff
@@ -97,6 +97,12 @@ https://management.azure.com/ for azure PublicCloud
 https://management.usgovcloudapi.net/ for azure USGovernmentCloud
 Users can also use this in case of a Hybrid Cloud model, where one may define their own audiences.
 
+`enable_batch_api` ::
+_boolean_
+Optional, by default is set to False. Set this to True when facing scalability issues. When configured, the azure batch api will be used
+to fetch metrics of multiple resources in one api call. 
+Currently supported metricsets are monitor, container_registry, container_instance, container_service, compute_vm, compute_vm_scaleset, database_account and storage.
```

Comment:
Can we also add `storage` to the list, or remove it because the metricset supports all the metricsets, right?

Reply:
Isn't storage in this list?